inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -0,0 +1,325 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import record_cite_modal, record_abstract with context %}
+
+{% macro record_collection_heading() %}
+	<span id="search-hep">Search HEP &gt;</span>
+	<span id="title">{{ record_title() }}</span>
+{% endmacro %}
+
+{% macro record_title() %}
+	{% block title %}
+		{% if record['title'] | is_list %}
+        	{% for title in record['title'] %}
+          	{{ title['title']|capitalize }}
+          {% endfor %}
+        {% else %}
+        	{{ record['title']['title']|capitalize }}
+        {% endif %}
+	{% endblock %}
+{% endmacro %}
+
+{% macro record_buttons() %}
+	<a class="btn btn-default dropdown-toggle dropdown-cite cite-btn" type="button" id="dropdownMenu{{record['control_number']}}" data-recid="{{record['control_number']}}"  data-toggle="modal" data-target="#citeModal{{record['control_number']}}">
+		<i class="fa fa-quote-left"></i> Cite this record
+	</a>
+	{% if record.get('url') %}
+		{% set exists = [] %}
+		{% for url in record.get('url') %}
```

Comment:
My tab width is set to 2 for TPL files (and appears correctly on my machine). Might need to check IRL.